README.md

```diff
@@ -9,9 +9,9 @@
 [![Lemmy](https://libre-tube.github.io/assets/widgets/lemmy-widget.svg)](https://feddit.rocks/c/libretube)
 
 </div>
-
-> **Note** <br>
-> We don't accept feature or bug requests on these platforms. Kindly submit requests only on GitHub.
+<div>
+**Note** <br>
```

Comment:
This change removes the bold formatting of "Note".

Reply:
@Bnyro Can you please check code once again and review it